charts/px-backup/templates/nginx-ingress-controller.yaml

```diff
@@ -0,0 +1,607 @@
+{{/* Setting defaults if they are omitted. */}}
+{{- $customRegistryEnabled := .Values.customeRegistryEnabled | default false }}
+{{- $ingressControllerSetupRequired := .Values.ingressControllerSetup | default false }}
+{{- if eq $ingressControllerSetupRequired true }}
+# Source: ingress-nginx/templates/controller-serviceaccount.yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    app.kubernetes.io/name: ingress-nginx
```

Comment:
These shouldn't be hardcoded. They need to be derived from the release.

Example of how we define these in the IBM helm charts: https://github.com/IBM/charts/blob/master/community/portworx/templates/stork-deployment.yaml#L9

The helper is defined here: https://github.com/IBM/charts/blob/master/community/portworx/templates/_helpers.tpl